app/views/works/home.html.erb

```diff
@@ -0,0 +1,66 @@
+<% if flash[:success] %>
+<div class="alert alert-info">
+    <%= flash[:success] %>
+</div>
+<% end %>
+
+<% if flash[:error] %>
+<div class="alert alert-danger">
+    <%= flash[:error] %>
+</div>
+<% end %>
+
+<div class="card border-0">
+    <div class="card-body">
+        <div class="card-body">
+            <% spotlight = Work.spotlight %>
+            <h2 class="card-title row bold-title">Media Spotlight: <div class="spotlight bold-title"> <%= link_to spotlight.title, work_path(spotlight), style:"text-decoration: underline" %> by <%= spotlight.creator %></div>
+            </h2>
+            <p class="card-text">
+                <p><%= spotlight.votes.count %> votes - <%= spotlight.description %></p>
+
+        </div>
+    </div>
+</div>
+
+<div class="row flex-nowrap">
```

Comment:
You essentially have the same code repeated three times. Consider how you could use a partial view or a loop to DRY up this code.